app/main/forms.py

```diff
@@ -1947,3 +1947,13 @@ class ChooseCoordinateTypeForm(StripWhitespaceForm):
         },
         validators=[DataRequired(message="Select which type of coordinates you'd like to use")],
     )
+
+
+class RejectionReasonForm(StripWhitespaceForm):
+    hint = """ Detailed reason for why the alert was rejected, including how it may be reworked.
```

Comment:
change to `Detailed reason why the alert...` or `Detailed reason for rejecting the alert,`